spack/packages/dla-future/package.py

```diff
@@ -138,6 +154,11 @@ def cmake_args(self):
                     " ".join([spec[dep].libs.ld_flags for dep in ["blas", "lapack"]]),
                 )
             )
+            if "+scalapack" in spec:
+                args.append(self.define("SCALAPACK_LIBRARY", spec["scalapack"].libs.ld_flags))
```

Comment:
Be careful here too with the name of the variable.

Reply:
This should be consistent with the `LAPACK_LIBRARY` variable above.